packages/pfring-kernel/PKGBUILD

```diff
@@ -0,0 +1,45 @@
+# Maintainer: h0bbl3s <david.frisbee@gmail.com>
+# Contributors: Tony Lambiris <tony@criticalstack.com>,
+#              David Manouchehri
+
+pkgver=7.0.bbaa482
+pkgrel=1
+_gitname=PF_RING
+pkgname="pfring-kernel"
+_gitbranch=dev
+_gitauthor=ntop
+pkgdesc="High-speed packet processing framework (dev branch) - kernel module"
+groups=('blackarch' 'blackarch-networking')
+url="https://github.com/${_gitauthor}/${_gitname}"
+license=('LGPL')
+source=("git://github.com/${_gitauthor}/${_gitname}#branch=${_gitbranch}")
+depends=('numactl')
+makedepends=('linux-headers' 'make' 'binutils' 'flex' 'bison' 'git')
+conflicts=()
+provides=("pfring-kernel")
+optdepends=()
+sha512sums=('SKIP')
+arch=('x86_64')
+
+            
+build() {
+	cd "${srcdir}/${_gitname}/kernel"
+	./configure --prefix=/usr
+	sed -i "s/INSTDIR\ \:=\ \$(DESTDIR)/INSTDIR := \$(DESTDIR)\/usr/g" Makefile
+	make
+}
+
+package() {
+	cd "${srcdir}/${_gitname}/kernel"
+	mkdir -p "${pkgdir}/usr/include/linux"
+	make DESTDIR="${pkgdir}" install;
+        mkdir -p "${pkgdir}/usr/lib/systemd/system"
+        mkdir -p "${pkgdir}/usr/local/bin"
+        cp "${srcdir}/${_gitname}/package/usr/local/bin/clusterctl" "${pkgdir}/usr/local/bin"
+        cp "${srcdir}/${_gitname}/package/usr/local/bin/pf_ringctl" "${pkgdir}/usr/local/bin/"
+        cp "${srcdir}/${_gitname}/package/etc/systemd/system/pf_ring.service" "${pkgdir}/usr/lib/systemd/system/"
+        cp "${srcdir}/${_gitname}/package/etc/systemd/system/cluster.service" "${pkgdir}/usr/lib/systemd/system/" 
+#        cp "${srcdir}/${_gitname}/package/etc/systemd/system/cluster@.service" "${pgkdir}/usr/lib/systemd/system/"
```

Comment:
Add the comment sign close to the beginning of the input, not the beginning of the line. It helps if I'm checking on a terminal without highlight syntax (kinda pedantic I know, but helps to avoid mistakes).

Reply:
Gotcha. Fixed. No problem I'm absolutely willing to learn the way you guys like things and why.